src/app/app.js

```diff
@@ -1,4 +1,6 @@
 // Special Debug Console Calls!
+var localStorage = window.localStorage || {};
```

Comment:
If you're 100% sure it won't destroy localStorage for the entire session if it fails to init, I don't see a problem, but from the outside, it looks like (if there's a fail) you replace the global variable by a volatile object that won't be saved on exit, hence that won't be available for next session. So maybe you should add in the try/catch of win.close() a `window.localStorage = localStorage` to make sure the modifications to localStorage are saved if localStorage is available AFTER the app started. Dunno if I make sense ? :p